apps/client/components/feature-modules/onboarding/components/forms/template-step-form.tsx

```
@@ -0,0 +1,229 @@
+      <div className="grid grid-cols-1 gap-3 sm:grid-cols-2">
+      {bundles.map((bundle) => {
+        const isSelected = selectedBundleKey === bundle.key;
+        const isExpanded = expandedBundleKey === bundle.key;
+        const matchedTemplates = bundle.templateKeys
+          .map((key) => templates.find((t) => t.key === key))
+          .filter(Boolean) as import('@/lib/types').ManifestSummary[];
+
+        return (
+          <div key={bundle.key} className="flex flex-col">
+            <div
+              role="button"
```

Biome (2.4.6)

[error] The elements with this role can be changed to the following elements:
<button>

(lint/a11y/useSemanticElements)